sdk/servicebus/Azure.Messaging.ServiceBus/tests/Administration/ServiceBusManagementClientLiveTests.cs

```diff
@@ -460,7 +460,7 @@ public async Task GetQueueRuntimeInfo()
 
             ServiceBusReceiver receiver = sbClient.CreateReceiver(queueName);
             ServiceBusReceivedMessage msg = await receiver.ReceiveMessageAsync();
-            await receiver.DeadLetterMessageAsync(msg.LockToken);
```

Comment:
I figured from the perspective of the tests it is better to go through the public API where possible so I switched all of them to use the public overload that accepts the message instead of switching them to `msg.LockTokenGuid`.